### DISCUSSION TITLE
How can I install pac tools on Linux?

### DISCUSSION THREAD

**Discussion by kurtschenk** [microsoft/powerplatform-build-tools#483] [Q&A]
I am unable to install "PD" or other tools.

# pac tool PD
Microsoft PowerPlatform CLI
Version: 1.25.5+g24fc1f8
Online documentation: https://aka.ms/PowerPlatformCLI
Feedback, Suggestions, Issues: https://github.com/microsoft/powerplatform-build-tools/discussions

Error: Not a valid command. Try running 'pac [command] help'.

==

But on Windows with PowerShell this works fine.

PS c:\<PATH> pac tool PD  
Installing 9.1.0.129 version of PD....
Shortcut in start menu created for 'Package Deployer'
Installation complete
Launched PD (9.1.0.129).

**Comment by Laskewitz** [comment]
Hi @KurtSchenk - 

The tools, like Package Deployer (PD), Configuration Migration Tool (CMT) and Plugin Registstration Tool (PRT) are Windows only applications, so they won't work on MacOS and Linux. The documentation page of the commands (https://aka.ms/pac/tool) shows a note that it only works for .NET full framework - which is only available for Windows.

Kind regards,
Daniel
